src/common/config_opts.h

```diff
@@ -261,6 +261,8 @@ OPTION(mon_health_to_clog_tick_interval, OPT_DOUBLE, 60.0)
 OPTION(mon_data_avail_crit, OPT_INT, 5)
 OPTION(mon_data_avail_warn, OPT_INT, 30)
 OPTION(mon_data_size_warn, OPT_U64, 15*1024*1024*1024) // issue a warning when the monitor's data store goes over 15GB (in bytes)
+OPTION(mon_warn_not_scrubbed, OPT_INT, 0)
```

Comment:
Why not make mon_warn_not_scrubbed mean how long past the mon_scrub_interval that the warning should appear?  So if you change the mon_scrub_interval you wouldn't necessarily have to modify the mon_warn_not_scrubbed value.

For consistency with mon_scrub_interval, mon_warn_not_scrubbed should be in seconds not minutes.

For example specified in seconds and relative to programmed interval.
If mon_scrub_interval changed from default (1 day) to 2 days
OPTION(mon_scrub_interval, OPT_INT, 3600*48) // every 2 days
If user set mon_warn_not_scrubbed  to 144000 (3600*4, 4 hours), it would warn if 52 hours have gone by since last scrub stamp.

It would be fine for the code to convert all values to minutes and do all checks in minutes even if value specified in seconds.

Same for mon_warn_not_deep_scrubbed.